components/Pages/EmployeePage/EmployeePage.tsx

```diff
@@ -0,0 +1,17 @@
+import useIsMobile from "../../../hooks/useIsMobile"
```

Comment:
optional: can you use `@/hooks` (absolute imports) if more than one level of`../` relative import?